src/NzbDrone.Core/Indexers/Definitions/BroadcastheNet/BroadcastheNetRequestGenerator.cs

```diff
@@ -50,7 +50,7 @@ public IndexerPageableRequestChain GetSearchRequests(TvSearchCriteria searchCrit
 
             var parameters = new BroadcastheNetTorrentQuery();
 
-            var searchString = searchCriteria.SearchTerm ?? "";
+            var searchString = searchCriteria.SearchTerm?.Trim() ?? "";
```

Comment:
Should we go ahead and do this in ReleaseSearchService so it happens for all queries?

Reply:
Yea, was just meaning the trim that was added. Something like this `spec.SearchTerm = query.q?.Trim();`